src/js/containers/Login.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+import { UserManager, Log, WebStorageStateStore } from 'oidc-client'
+//import { redirectAuth, processAuth } from '../actions'
+import Login from '../components/Login.jsx'
+import userManager from '../UserManager.js'
+
+
+class LoginContainer extends Component {
+  constructor(props) {
+      super(props)
+  }
+
+  componentDidMount() {
+    const manager = new UserManager({
+      authority: 'https://192.168.99.100:8443/auth/realms/hmda',
```

Comment:
Same `UserManager` question as above.  This one does seem a bit different, though.  For instance, there's no _silent renew_-related attributes.  Is that intentional?  What's the difference between these two `UserManager`s?

When we do change this, could we create a JSON file that gets read in instead so we don't have to find/replace in-code?  Also, then it'd be setup in a single place.